src/includes/getting-started-primer/javascript.nextjs.mdx

```diff
@@ -1,10 +1,10 @@
 <Note>
 
-Sentry's Next.js SDK enables automatic reporting of errors and exceptions.
+Currently, the Sentry Next.js SDK supports Next.js versions `10.0.8` - `11.1.3`. The SDK may work with some projects using Next.js 12, but it is not yet officially supported.
 
 </Note>
 
-Currently, the minimum Next.js supported version is `10.0.8`.
+The Sentry Next.js SDK automatically catches errors and collects performance data from your app.
```

Comment:
I agree having what the SDK is able to do in a note doesn't look like the best option, but I don't think we should change it unless we also make the change for all the other platforms and products.

Reply:
It's actually inconsistent - some frameworks have it (Angular, for example) and some don't (Gatsby, Electron, etc).

Reply:
Mmm... Then not blocking.